crates/node-services/proto/schema.proto

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2024 - Restate Software, Inc., Restate GmbH
+//
+// This file is part of the Restate service protocol, which is
+// released under the MIT license.
+//
+// You can find a copy of the license in file LICENSE in the root
+// directory of this repository or package, or at
+// https://github.com/restatedev/proto/blob/main/LICENSE
+
+syntax = "proto3";
+
+import "google/protobuf/empty.proto";
+
+package dev.restate.schema;
+
+service Schema {
```

Comment:
Maybe this can be the seed for the Metadata service? I also think we should postfix all grpc services with something to make it distinct from potential message types, perhaps with `Svc` at the end or something.

Note: The postfix comment isn't related to this particular PR.

Reply:
Yes, I thought so too concerning the `Metadata` service. Should we start right away calling it `MetadataSvc` instead? Given that these services are currently only internally used, it is probably also possible to rename them later.

Re postfix: +1 will change it.